### DISCUSSION TITLE
Immediate Upated Required pls help

### DISCUSSION THREAD

**Discussion by that1kidd0** [3kh0/ext-remover#578] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
AFter downgrading i just go so mad that it says that screen pls help me

**Comment by AdanGonz2444** [comment, marked as answer]
Uhm, I do not think that made much sense?

**Reply by srets** [reply]
he said after he downgraded, the screen just only said immediate update required
